tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "es2015",
     "module": "commonjs",
     "moduleResolution": "node",
-    "lib": ["es2018"],
+    "lib": ["es2018", "dom"],
```

Comment:
Since the client lib is now in here, we need this so fetch is available to it. DOM stuff shouldn't be used in server, but I think this is a pretty meager sacrifice.